test/test_proxy.py

```diff
@@ -0,0 +1,371 @@
+from datetime import datetime
+import unittest
+import json
+from unittest import mock
```

Comment:
Related to the other comment, move this along `proxy.py`.
The `test/` directory is where all tests + registry tests used to go . Since then we've favoured having tests being stored along side its module
If you look some other module, say `util/secscan/`, the secscan tests are stored in `util/secscan/test`. It's just easier for discovery and navigating specific tests this way.

Reply:
Answered in [previous comment](https://github.com/quay/quay/pull/1053#discussion_r800455617)